### ISSUE TITLE
Frontend: Overview of digital identities

### ISSUE THREAD

**Issue by TamaraHoock** [amosproj/amos2022ss04-digital-identity#10]
## User story
1. As a user (HR employee)
2. I want / need an overview of all digital identieties and open processes
3. So that I can check which digital identities exist 

## Acceptance criteria
* Table overview as provided by mockup (e.g. Column with status)
* edit-button which will link to editing page in the future
* no need to display data from real backend at the moment. Dummy values can be displayed

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Idontker** [comment]
Finished in sprint 4:  b0d9955e98f9a9708c55925ad923adabc86005db
